__tests__/app.test.js

```diff
@@ -36,3 +36,43 @@ describe('/api/topics', () => {
             })
     })
 })
+
+describe('/api/articles/:article_id', () => {
+    it('GET - 200 - Returns the article with ID provided', () => {
+        return request(app).get('/api/articles/' + 1)
+            .expect(200)
+            .then(({ body: { article } }) => {
+                expect(article).toEqual({
```

Comment:
toEqual does work here, but it doesn't allow for extension, for example, if you add another property to this endpoint. Have a look at `.toMatchObject` as it would be ideal here